docs/book/src/developer/tilt.md

```diff
@@ -378,6 +378,55 @@ Set to `false` if your provider does not have a ./config folder or you do not wa
 in tilt version >= v0.22.2 (see https://blog.tilt.dev/2021/08/09/resource-grouping.html); as a convention,
 provider abbreviation should be used (CAPD, KCP etc.).
```

Comment:
(unrelated to this filed just had to add it somewhere)

We're adding a Tips & Tricks section to the implementation doc in https://github.com/kubernetes-sigs/cluster-api/pull/7270.

There was a suggestion to also mention the tilt setup there:
> What if we add a note about being possible to develop and test with tilt (assuming that https://github.com/kubernetes-sigs/cluster-api/pull/7154 merge soon, this note could close the circle)
> https://github.com/kubernetes-sigs/cluster-api/pull/7270#discussion_r977565818